v2/internal/pkg/spinners/spinners.go

```diff
@@ -3,6 +3,7 @@ package spinners
 import (
 	"io"
 
+	"github.com/openshift/oc-mirror/v2/internal/pkg/emoji"
```

Comment:
```suggestion
        "io"

	"github.com/vbauerster/mpb/v8"
	"github.com/vbauerster/mpb/v8/decor"

	"github.com/openshift/oc-mirror/v2/internal/pkg/emoji"
```

To fix linting issue